### PR TITLE
Adding IcingaStatusWriter object. #7230

### PR DESCRIPTION
**Pull request by cropalato**
https://dev.icinga.org/issues/7230

### CODE REVIEW

File: manifests/object/icingastatuswriter.pp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# http://docs.icinga.org/icinga2/latest/doc/module/icinga2/chapter/configuring-icinga2#objecttype-checkercomponent
+#
+# === Parameters
+#
+# See the inline comments.
+#
+
+define icinga2::object::icingastatuswriter (
+  $ensure                    = 'file',
+  $object_name               = $name,
```
Comment (nickchappell) [inline manifests/object/icingastatuswriter.pp:14]: You'll have to set `$target_file_name` to `$name`. The `$name` variable is available and defined when Puppet evaluates the defined type, but it looks like the `$object_name` variable doesn't get 'set' internally until after it tries to create the file resource. When I tested it, it created the file as `/etc/icinga2/conf.d/.conf`.

**Comment by cropalato** [comment]
Hi @nickchappell, 
I did'n face the same problem than you. But I found a typo and I fixed. thanks.

**Comment by nickchappell** [comment]
Just wanted to give you a heads up that I'll be traveling with family for a little while and may not able to review/merge this until after Christmas.

**Comment by nickchappell** [comment]
I merged this into the develop branch. Thanks @cropalato!

**Pull request closed without merging**